### PR TITLE
add core members for the ipfs docs

### PR DESCRIPTION
**Pull request by jennijuju**

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Review by 2color — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -145,8 +145,11 @@ Please stay tuned on the steps to translate the documentation.
 
 ## Core members
```
Comment (DannyS03) [inline README.md:146]:
```suggestion
## Primary maintainers
```

File: README.md [hunk 2]
```diff
@@ -145,8 +145,11 @@ Please stay tuned on the steps to translate the documentation.
...
 
-- [@johnnymatthews](https://github.com/johnnymatthews): Project leadership, organization, and primary contact
+- [@TMoMoreau](https://github.com/TMoMoreau): Primary contact, tech writer(engineering)
```
Comment (DannyS03) [inline README.md:148]:
```suggestion
- [@TMoMoreau](https://github.com/TMoMoreau): Primary contact, technical writing(engineering)
```

File: README.md [hunk 3]
```diff
@@ -145,8 +145,11 @@ Please stay tuned on the steps to translate the documentation.
...
+- [@johnnymatthews](https://github.com/johnnymatthews): Project leadership, organization
```
Comment (DannyS03) [inline README.md:149]:
```suggestion
- [@johnnymatthews](https://github.com/johnnymatthews): Project leadership & organization
```

File: README.md [hunk 4]
```diff
@@ -145,8 +145,11 @@ Please stay tuned on the steps to translate the documentation.
...
 - [@cwaring](https://github.com/cwaring): Development support
+- [@2color](https://github.com/2color): Dev Rel, tech writer(ecosystem)
```
Comment (DannyS03) [inline README.md:151]:
```suggestion
- [@2color](https://github.com/2color): Developer relations & technical writing(ecosystem)
- [@dannyS03](https://github.com/DannyS03}: Technical writing(engineering) 
```

File: README.md [hunk 5]
```diff
@@ -145,8 +145,11 @@ Please stay tuned on the steps to translate the documentation.
...
+- [@jennijuju](https://github.com/jennijuju): TPM
```
Comment (DannyS03) [inline README.md:152]:
```suggestion
- [@jennijuju](https://github.com/jennijuju): Management and supervision
```

**Review by DannyS03 — CHANGES_REQUESTED** [review]

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Review by DannyS03 — APPROVED** [review]

**Pull request merged**